types/js-worker-search/tslint.json

```diff
@@ -0,0 +1,17 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "npm-naming": [
```

Comment:
It looks like this wasn't actually removed as you intended. Could you please remove it? Thanks!

Reply:
I removed it, but then Travis CI complained that I needed to put it back in -- because of the way that imports are set up in js-worker-search. I can get more info if you need, but it was a frustrating situation where I'd submit one way and get dinged for it, then submit the other way and get dinged for that....

Reply:
npm-naming is getting confused by the webpack output that js-worker-search ships. So you'll need to keep the rule exemption.